components/post_view/post_list.jsx

```diff
@@ -251,6 +251,7 @@ export default class PostList extends React.PureComponent {
     }
 
     loadMorePosts = async () => {
+        this.loadingMorePosts = true;
```

Comment:
This is the only change required for the PR. made other var name changes for readability